README.md

```diff
@@ -120,6 +122,21 @@ The script also supports branches automatically. If you work in a checked out br
 
 Preview mode works in a feature branch, apply script which creates new preview branch and create additional commit with customization.
 
+### Bootstrapping with Preview modes
+
+Two preview modes are offered:
+
+- `preview-ckcp` deploys containerized KCP directly into your cluster and connects the cluster.
+- `preview-cps` connects to CPS instance. Before the mode is used the kubeconfig file must be created manually based on 'CSP onboarding document' and set `CPS_KUBECONFIG` variable in `hack/preview.env`
```

Comment:
CPS_KUBECONFIG is copied to KCP_KUBECONFIG, KCP_KUBECONFIG is managed by preview mode and CPS_KUBECONFIG is only source which is not touched

Reply:
Ok, now I understand what is your approach here - you don't expect that KCP_KUBECONFIG would be set to any already existing kubeconfig (as we for example do in upstream or dev mode), because you just override it with what you need in the particular mode (ckcp/cps).
Wouldn't it be safer to use some `/tmp/` file for such "copy" operations? Just asking ;-)